workflows/prognostic_c48_run/runtime/diagnostics/compute.py

```diff
@@ -24,8 +24,9 @@ def enforce_heating_and_moistening_tendency_constraints(
     timestep: float,
     hydrostatic: bool,
     mse_conserving: bool,
-    temperature_tendency_name="dQ1",
-    humidity_tendency_name="dQ2",
+    temperature_tendency_name: str = "dQ1",
+    humidity_tendency_name: str = "dQ2",
+    zero_fill_missing_tendencies: bool = False,
```

Comment:
This is added as an optional arg to preserve prior behavior in the ML postphysics stepper where these diags aren't output if dQ1, dQ2 are not present. If this isn't important I can remove it and the zero-filled fields will always be in the diags.

Reply:
👍 I think it's best to preserve previous behavior when possible.